src/Sarif/RuntimeConditions.cs

```diff
@@ -49,9 +50,11 @@ public enum RuntimeConditions : uint
         ExceptionInstantiatingSkimmers = 0x8000,
         OutputFileAlreadyExists = 0x10000,
         ExceptionProcessingBaseline = 0x20000,
+        ExceptionExportingLogFile = 0x40000,
```

Comment:
>ExceptionExportingLogFile [](http://example.com/codeflow?start=8&length=25)

ExceptionPostingLog

Reply:
renamed! :)

Reply:
Actually this needs to be ExceptionPostingLogFile, see ExitReason.cs, we should stay in sync with that casing. You can merge after making this change.